### PR TITLE
Add example using variant id in capture payload

### PR DESCRIPTION
**Pull request by jaepass**

### CODE REVIEW

**Review by robbieaverill — COMMENTED** [review]
The indentation in your example is a bit wonky. Can you please make sure it's all 2 spaces consistently? Also, the way we're writing JavaScript objects in these examples is a bit weird, it looks like we've just copy and pasted some JSON. We should fix that. This particular example can probably have the billing and fulfillment objects removed since they're already in the previous examples and they're mostly wasting space in this example since it's about the variant_id argument.

**Comment by jaepass** [comment]
I'll get rid of the quotes and maybe should replace 'token' with an example token id

File: docs/03_Checkout.md [hunk 1]
```diff
@@ -126,7 +126,8 @@ submitting data. For example with line items, the key would be the `line_item_id
 under that key.
 
 * Line item's quantity: `line_items[{line_item_id}][quantity]`
-* Line item's variant and selected option: `line_items[{line_item_id}][variants][{group_id}] = {option_id}`
+* Line item's variant group and selected option: `line_items[{line_item_id}][variants][{group_id}] = {option_id}` *OR*
+* Line items's requested variant id: `line_items[{line_item_id}][variant_id][{variant_id}]`
```
Comment (robbieaverill) [inline docs/03_Checkout.md:130]:
```suggestion
* Line items's requested variant id: `line_items[{line_item_id}][variant_id] = {variant_id}`
```

**Review by robbieaverill — APPROVED** [review]
Our JavaScript linter would also encourage single quotes over double quotes, but again not something that needs to block merging this PR

**Comment by jaepass** [comment]
I've change to single quotes since should also be consistent with all other examples in this doc

**Pull request merged**